src/app/components/Grid/index.jsx

```diff
@@ -21,4 +23,138 @@ export const GelPageGrid = styled(Grid)`
   ${gelMaxWidths}
 `;
 
+/* This is intented to be used on pages with single columns
+ * i.e. ArticlePage, PhotoGalleryPage and MediaAssetPage
+ */
+export const SingleColumnWrapper = styled.div`
```

Comment:
Please can you help me understand how this component will be used, Isaac? Is it designed to wrap `GelPageGrid` on the listed page types? An example would be super-helpful.

Reply:
@simonsinclair I have added a storybook example.

Reply:
Thanks for providing an example, Isaac. I will re-review.